guides/source/getting_started.md

```diff
@@ -605,9 +597,11 @@ class ArticlesController < ApplicationController
 end
 ```
 
-If you re-submit the form now, you'll see another familiar error: a template is
-missing. That's ok, we can ignore that for now. What the `create` action should
-be doing is saving our new article to the database.
+If you re-submit the form now, you may not see any change on the page. Don't worry!
+This is because Rails by default returns `204 No Content` response for an action if
+we don't specify how the response should be. We just added the `create` action
+but didn't specify anything about how the response should be. In this case, what the
+`create` action should be doing is saving our new article to the database.
```

Comment:
Part of these sentences read a bit off. Can you update these as follows? Thanks!

"specify how the response should be" -> "specify what the response should be."

"In this case, what the `create` action should be doing is saving our new article to the database." -> "In this case, the `create` action should save our new article to the database."